spec.emu

```diff
@@ -1006,6 +1006,19 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
         BindingIdentifier[~Yield, +Await]
     </emu-grammar>
 
+    <emu-note>
+      <p>|LineTerminator| is disallowed between the `source` and |ImportedBinding| for forward compatibility with the <a href="https://github.com/tc39/proposal-module-declarations">module declarations proposal</a>.</p>
```

Comment:
I don't think we even need to mention compatibility here as a note? Probably best not to forward reference spec dependence too much.